pyuploadcare/api_resources.py

```diff
@@ -232,23 +232,85 @@ def store(self):
+                      create_remote_copy instead. """
+        logger.warn('API Warning: {0}'.format(warning))
+
+        if target is not None:
+             return self.create_remote_copy(target, effects)
+        else:
+             return self.create_local_copy(effects)
+
+    def create_local_copy(self, effects=None, store=None):
+        """Creates a Local File Copy on Uploadcare Storage
+
+        - effects:
+            Adds CDN image effects. If ``self.default_effects`` property is set
+            effects will be combined with default effects.
+        - store:
+            If ``store`` option is set to False the copy of your file will be deleted
+            in 24 hour period after the upload.
         """
         effects = effects or ''
+        store = store or ''
         if self.default_effects is not None:
             fmt = '{head}-/{tail}' if effects else '{head}'
             effects = fmt.format(head=self.default_effects,
                                  tail=effects)
         data = {
             'source': self.cdn_path(effects)
         }
-        if target is not None:
-            data['target'] = target
+        if store is not None:
+            data['store'] = store
+        return rest_request('POST', 'files/', data=data)
+
+    def create_remote_copy(self, target, effects=None, make_public=None, pattern=None):
+        """Creates File Copy On Remote Storage
+
+        - target:
+            Target name associated with the project;
+        - effects:
+            Adds CDN image effects to ``self.default_effects`` if any;
+        - make_public:
+            To forbid public from accessing your files on the storage set
+            ``make_public`` option to be False.
+            Default value is None. Files have public access by default.
+        - pattern:
+            Specify ``pattern`` option to set S3 object key name. Takes precedence over
+            pattern set in project settings. If neither is specified defaults to
+            ${uuid}/${filename} ${effects} ${ext}
+
+        For more information on each of the options above please refer to
+        our docs https://uploadcare.com/documentation/rest/#file.
+
+        Following example copies a file to custom storage named ``samplefs``:
+             >>>file = File('e8ebfe20-8c11-4a94-9b40-52ecad7d8d1a');
+             >>>file.remote_copy(target='samplefs',
+                                 make_public=True,
+                                 pattern='${uuid}/${filename}${ext}');
+        Now custom storage ``samplefs`` contains publicly available file
+        with original filename billmurray.jpg in
+        in the directory named ``e8ebfe20-8c11-4a94-9b40-52ecad7d8d1a``.
+
+
+        """
+        effects = effects or ''
+        if self.default_effects is not None:
+            fmt = '{head}-/{tail}' if effects else '{head}'
+            effects = fmt.format(head=self.default_effects,
+                                 tail=effects)
```

Comment:
It seems there is copy of this code here https://github.com/uploadcare/pyuploadcare/pull/116/files#diff-2e1fdb139bd11faec8328901cf61df9aL242
I suggest making a method from this code